lib/plugins/bodyParser.js

```diff
@@ -39,6 +39,13 @@ function bodyParser(options) {
     var parseFieldedText = fieldedTextParser(opts);
 
     function parseBody(req, res, next) {
+        // #100 don't parse the body again if we've read it once
+        if (req._parsedBody) {
+            next();
```

Comment:
We do, but this is also scope creep as I'm just being consistent with the rest of the repo. Happy to discuss in a separate issue.